sdks/java/harness/src/main/java/org/apache/beam/fn/harness/data/ElementCountFnDataReceiver.java

```diff
@@ -35,20 +39,30 @@
 
   private FnDataReceiver<WindowedValue<T>> original;
   private Counter counter;
+  private MetricsContainer unboundMetricContainer;
 
-  public ElementCountFnDataReceiver(FnDataReceiver<WindowedValue<T>> original, String pCollection) {
+  public ElementCountFnDataReceiver(
+      FnDataReceiver<WindowedValue<T>> original,
+      String pCollection,
+      MetricsContainerStepMap metricContainerRegistry) {
     this.original = original;
     HashMap<String, String> labels = new HashMap<String, String>();
     labels.put(SimpleMonitoringInfoBuilder.PCOLLECTION_LABEL, pCollection);
     MonitoringInfoMetricName metricName =
         MonitoringInfoMetricName.named(SimpleMonitoringInfoBuilder.ELEMENT_COUNT_URN, labels);
     this.counter = LabeledMetrics.counter(metricName);
+    // Place it in a metric container which is not bound to the step name.
```

Comment:
I'm having trouble parsing this comment. What is "it"? "not likely" below is a bit cryptic… can it ever happen?

If you are moved to elaborate, that would be great, but not essential 😀